tests/conftest.py

```diff
@@ -17,3 +19,8 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
+
+
+def is_on_slurm() -> bool:
```

Comment:
@rmudambi  I'd say this should be something we import from build_utils but I'm putting it here so builds pass for now.